templates/repo/branch/list.tmpl

```diff
@@ -1,4 +1,9 @@
 {{template "base/head" .}}
+<script>
+	window.config.i18n_branch = {
+		rename_branch_to: '{{.locale.Tr "repo.branch.rename_branch_to"}}',
```

Comment:
Why there was no such i18n key before, but we need it now?

Reply:
Before, the function of renaming branch is in the page of repositroy settings. Users need to fill in both `old branch name` and `new branch name` by themself.
Now, this function move to the page of branches list, user just need to select the branch which they want to rename, and then fill in the `new branch name` in the dialog.
Although user don't need to fill in the `old branch name`, I think we need to prompt the user for the `old branch name` to be changed in the dialog. And the value of `old branch name` is not determined until the click event is triggered.  So add this i18n key in javascript.
![image](https://user-images.githubusercontent.com/33891828/235061913-a33a0d11-fdb2-48a6-ad6c-88fa4d812808.png)

Reply:
You can pass the i18n string by `attr` instead of making the global JS object too complex

Reply:
8a77aefb5967fe853d4dba86ba7aaa02c75fb021